solr/core/src/java/org/apache/solr/search/CollapsingQParserPlugin.java

```diff
@@ -188,6 +206,11 @@ public int hashCode() {
       return 17 * (31 + selectorText.hashCode()) * (31 + type.hashCode());
     }
 
+    @Override
+    public String toString(){
+      return "groupHeadSelectorText=" +this.selectorText + ", groupHeadSelectorType=" +this.type;
```

Comment:
would it be better to include the class name too here?
```suggestion
      return "GroupHeadSelector[selectorText=" +this.selectorText + ", type=" +this.type + "]";
```